src/include/log.h

```diff
@@ -92,6 +118,12 @@ union __wt_lsn {
  * a few special states, reserve the top few bits for state.  That makes
  * the maximum size less than 32 bits for both joined and released.
  */
+/*
+ * XXX
```

Comment:
I'd prefer if this had a more obvious name like `FLD_LOG_SLOT_ISSET` since it is doing something a bit odd.

Reply:
> I'd prefer if this had a more obvious name like `FLD_LOG_SLOT_ISSET` since it is doing something a bit odd.

Agreed, pushed in a3d72e4.